### PR TITLE
Update/dependencies

### PR DESCRIPTION
**Pull request by yubiuser**
Updates 

```
librespot
/workspaces/librespot/Cargo.toml
dependency  current  upgrade
sysinfo     0.31.4   0.33.0
thiserror   1.0.69   2.0.7

librespot-audio
/workspaces/librespot/audio/Cargo.toml
dependency  current  upgrade
thiserror   1.0.69   2.0.7

librespot-core
/workspaces/librespot/core/Cargo.toml
dependency  current  upgrade
governor    0.6.3    0.8.0
quick-xml   0.36.2   0.37.1
sysinfo     0.31.4   0.33.0
thiserror   1.0.69   2.0.7

librespot-oauth
/workspaces/librespot/oauth/Cargo.toml
dependency  current  upgrade
thiserror   1.0.69   2.0.7

librespot-connect
/workspaces/librespot/connect/Cargo.toml
dependency  current  upgrade
thiserror   1.0.69   2.0.7

librespot-playback
/workspaces/librespot/playback/Cargo.toml
dependency  current  upgrade
rodio       0.19.0   0.20.1
thiserror   1.0.69   2.0.7
zerocopy    0.7.35   0.8.13

librespot-metadata
/workspaces/librespot/metadata/Cargo.toml
dependency  current  upgrade
thiserror   1.0.69   2.0.7

librespot-discovery
/workspaces/librespot/discovery/Cargo.toml
dependency  current  upgrade
thiserror   1.0.69   2.0.7
```

### CODE REVIEW

**Comment by yubiuser** [comment]
I did not update `jack` and `zbus` as both require MSRV 1.80.0

File: core/src/session.rs [hunk 1]
```diff
@@ -390,7 +390,7 @@ impl Session {
     }
 
     pub fn set_session_id(&self, session_id: String) {
```
Comment (photovoltex) [inline core/src/session.rs:392]: Could you additionally change the type to `&str` and check where the method is used and remove possible clones?

**Comment by roderickvd** [comment]
👌

File: core/src/session.rs (hunk 1 above)
Comment (roderickvd) [reply core/src/session.rs:392]: Dang, missed that comment just as I merged. Please submit as an additional PR - sorry and thanks!

Comment (photovoltex) [reply core/src/session.rs:392]: Yeah was a bit to slow there^^;

Comment (yubiuser) [reply core/src/session.rs:392]: https://github.com/librespot-org/librespot/pull/1422

**Pull request merged**